apps/mobile/src/components/headers/components/header-title-with-subtitle.tsx

```diff
@@ -0,0 +1,22 @@
+import { Stack, Text } from '@leather.io/ui/native';
+
+export function HeaderTitleWithSubtitle({
+  title,
+  subtitle,
+  testID,
+}: {
+  title: string;
+  subtitle: string;
+  testID?: string;
+}) {
```

Comment:
👀 separate props interface?

Reply:
Not sure how I missed, thx. I'll change all the headers to use interfaces so this mistake can't be made again. Seeing lots that need interfaces added, so will try to do here.